source/manual/development-pipeline.html.md

```diff
@@ -58,6 +58,8 @@ When a deploy freeze is in effect, you should avoid merging any PRs. This is bec
 
 We've got [guidelines on merging of Pull Requests](/manual/merge-pr.html).
 
+> Some applications [have restrictions on who can merge PRs](https://github.com/alphagov/govuk-saas-config/blob/master/github/repo_overrides.yml). If you are unable to merge your own PR, you should ask someone else to merge (and deploy) it for you.
```

Comment:
For the `you should ask someone else to merge` - could we add how they find the right person to ask?

Reply:
That's a good idea. I've updated with a direct link to our production GitHub team.